specification/advisor/resource-manager/readme.md

```diff
@@ -89,6 +89,7 @@ python:
   payload-flattening-threshold: 2
   namespace: azure.mgmt.advisor
   package-name: azure-mgmt-advisor
+  package-version: 1.0.1

```

Comment:
@lmazuel cool?

Reply:
@olydis yes! I reviewed these changes offline, with actual RestAPI look. This is indeed an expected bugfix.